frontend/src/crosswordView/crosswordView.js

```diff
@@ -0,0 +1,343 @@
+                        <li key={k}>
+                            { definition }
+                        </li>
+                    ))}
+                </ul>
+            );
+        default:
+            return '';
+        }
+    }
+
+    modalHandler = () => {
+        this.setState({ showModal: !this.state.showModal });
+    }
+
+    render() {
+        if (!this.state.crosswordData) {
+            return (<LoadingPage />);
+        }
+
+        const {
+            found,
+            grid,
+            showDifficulty,
+            showModal,
+        } = this.state;
+
+        const clueBox = <div className='clues-box'>
+            <h2 className='clue-header'>Across:</h2>
+            {
+                this.state.crosswordData.clues.map((clue, k) => {
+                    if (clue.across) {
+                        return (
+                            <div
+                                className={`clue ${found[k].across ? 'clue-found' : ''}`}
+                                key={k}
+                            >
+                                {k + 1}) {found[k].across
+                                    ? clue.across.clue
+                                    : coverWord(clue.across.word, clue.across.clue)
+                                }
+                            </div>
+                        );
+                    }
+                    return '';
+                })
+            }
+            <h2 className='clue-header'>Down:</h2>
+            {
+                this.state.crosswordData.clues.map((clue, k) => {
+                    if (clue.down) {
+                        return (
+                            <div
+                                className={`clue ${found[k].down ? 'clue-found' : ''}`}
+                                key={k}
+                            >
+                                {k + 1}) {found[k].down
+                                    ? clue.down.clue
+                                    : coverWord(clue.down.word, clue.down.clue)
+                                }
+                            </div>
+                        );
+                    }
+                    return '';
+                })
+            }
+        </div>;
+
+        const definitionBox = (<div className='clues-box'>
+            <h2 className='clue-header'>Definitions</h2>
+            { this.getDefinitions() }
+        </div>);
+
+        const instructionModal = (<div>
+            {
+                showModal
+                    ? <div className="backdrop" onClick={this.modalHandler}>
+                    </div>
+                    : null
+            }
+            <div className="Modal modal-content" style={{
+                transform: showModal ? 'translateY(0)' : 'translateY(-100vh)',
+                opacity: showModal ? 1 : 0,
+            }}>
+                <div className="modal-header">
+                    <h5 className="modal-title">Instructions</h5>
```

Comment:
Please see my other comment about semantic HTML. The use of the `<h5>` here violates the [guidelines around HTML headers](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes) since it doesn't follow in the appropriate header sequence (i.e., sequentially, `<h1>`, `<h2>`, etc.).